cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -94,15 +96,24 @@
     }
 
     & > .cf-icon-svg {
-        // Ensuring play icon optically appears horizontally centered.
-        position: relative;
-        left: 0.09375em;
-
-        // Ensuring play icon is vertically centered.
-        vertical-align: bottom;
+        // Size and position visually centers the play arrow.
+        position: absolute;
+        left: 55%;
+        top: 50%;
+        transform: translate( -50%, -50% );
```

Comment:
Any particular reason for changing the positioning method? It seems to end up in the same place, but it takes one more rule.

Reply:
The other positioning didn't work the same for both a `a` link and `button`. This way it does work the same for both.